src/servers/database.ts

```diff
@@ -745,6 +769,15 @@ export class DatabaseServer extends SkeletonServer {
     return found;
   }
 
+  async eth_unsubscribe(
+    _request: Request,
+    _subsciptionId: web3.Data
+  ): Promise<boolean> {
+    const query = sql.delete('subscription').where({ 'id': _subsciptionId })
+    await this._query(query.toString());
```

Comment:
Generally better to use the `query.toParams()` form instead of `query.toString()`.